tests/integration/targets/setup_postgresql_db/tasks/main.yml

```diff
@@ -127,14 +127,16 @@
     group: '{{ pg_group }}'
     mode: '0644'
 
-- name: Generate locales (Debian)
-  locale_gen:
-    name: '{{ item }}'
-    state: present
-  with_items:
-  - pt_BR
-  - es_ES
-  when: ansible_os_family == 'Debian'
+### Comment: This block caused the tests to crash
+
+#- name: Generate locales (Debian)
```

Comment:
```suggestion
### Comment: This block caused the tests to crash

#- name: Generate locales (Debian)
```
Do you run them with --docker ?, e.g:
```
ansible-test integration <target_name> --docker ubuntu2004 -vvv > ~/test.log
```
i run the tests today and they were green

Reply:
I ran them exactly like that and they crashed with:
```
ERROR! couldn't resolve module/action 'locale_gen'. This often indicates a misspelling, missing collection, or incorrect module path.

The error appears to be in '/root/ansible_collections/community/postgresql/tests/output/.tmp/integration/postgresql_table-l3c1vxf2-ÅÑŚÌβŁÈ/tests/integration/targets/setup_postgresql_db/tasks/main.yml': line 132, column 3, but may
be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:


- name: Generate locales (Debian)
  ^ here
```
Also the apparently nothing breaks when the block is out commented.

[test.log](https://github.com/ansible-collections/community.postgresql/files/10390702/test.log)

Reply:
@SiL3x i've just checked out your branch, uncommented this block and run the tests locally:
```
$ ansible-test integration postgresql_table --docker ubuntu2004 -vvv > ~/test.log
WARNING: A Python 3.8 interpreter was found at "/usr/bin/python3.8", yet a matching pip was not found.
[WARNING]: running playbook inside collection community.postgresql
[WARNING]: Collection community.general does not support Ansible version 2.10.6
[WARNING]: Updating cache and auto-installing missing dependency: python3-apt
[WARNING]: Consider using the get_url or uri module rather than running 'wget'.
If you need to use command because get_url or uri is insufficient you can add
'warn: false' to this command task or set 'command_warnings=False' in
ansible.cfg to get rid of this message.
[WARNING]: Module remote_tmp /var/lib/postgresql/.ansible/tmp did not exist and
was created with a mode of 0700, this may cause issues when running as another
user. To avoid this, create the remote_tmp dir with the correct permissions
manually
[WARNING]: The value "True" (type bool) was converted to "'True'" (type
string). If this does not look like what you expect, quote the entire value to
ensure it does not change.
WARNING: Reviewing previous 1 warning(s):
WARNING: A Python 3.8 interpreter was found at "/usr/bin/python3.8", yet a matching pip was not found.
```
They pass.
Could you please uncomment the block and push? Let's see what our CI will say.